verification/verify.go

```diff
@@ -153,3 +163,80 @@ func IsRetryable(err error) bool {
 	_, retryable := err.(Retryable)
 	return retryable
 }
+
+func verifyPixelParams(params *Params) error {
+	// check if the node is on-chain mode
+	if params != nil &&
+		params.Orchestrator != nil &&
+		params.Orchestrator.TicketParams != nil {
+
+		var err error
+		for i := 0; err == nil && i < len(params.Results.Segments); i++ {
+			if err := verifyPixels(
+				params.Results.Segments[i].Url,
+				flatten(params.Renditions),
```

Comment:
We want to pass in `params.Renditions[i]` here so that in the case where in-memory storage is used, `params.Renditions[i]` is the data corresponding to `params.Results.Segments[i].Url`.

Reply:
done in c2c6653cd125ecae82c816ce49fbd66a9bf75f29